pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -149,7 +149,7 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 		// If the crash controller is disabled in the spec let's do a noop
 		if cephCluster.Spec.CrashCollector.Disable {
```

Comment:
THis will just disable the crash collector from the cluster, but not solve the actual problem, right

Reply:
The other change below (line 216) is more relevant to my problem. But I think line 152 needs to be changed too, in order to fix the issue in case one of the clusters has crash collector disabled and one has it enabled.

Reply:
Can you please elaborate on what different values you are getting on `namespace` and `request.namespace`

Reply:
I didn't test this, yet. But given that `namespace` changes in every loop iteration (of the loop in line 132) and `request` is constant, they will differ in allmost all iterations.

Reply:
That make sense

Reply:
Looking through this as well, agreed that the fix appears to be changing to use `namespace` since that's where the ceph pods were found that need to be removed. The `request.Namespace` is not applicable since it's from the node which doesn't even have a namespace.